CONTRIBUTING.md

```diff
@@ -21,6 +21,12 @@ consider this the window for comments.
 * All tests must pass on Travis CI for the merge to occur.
 * All changes must not introduce superfluous changes or whitespace errors.
 * All changes should adhere to the coding standard.
+* All commits should adhere to the git commit message guidlines described
+here: https://chris.beams.io/posts/git-commit/ with the following exceptions.
```

Comment:
Maybe it's better to refer to https://www.kernel.org/doc/html/latest/process/submitting-patches.html#the-canonical-patch-format which is a more formal and summarized version of what's mentioned in that article?

Or do you have any issues with the canonical patch format used by Linux?

Reply:
I ran across both but the `canonical-patch-format` includes a lot more data than just commit message format and describes everything in the context of sending patches to a mailing list. I figured including all of that would probably confuse folks.

Reply:
@flihp fair enough.